day_two_guide/topics/proc_restoring-etcd.adoc

```diff
@@ -11,253 +11,37 @@ Module included in the following assemblies:
 [id='restoring-etcd_{context}']
 = Restoring etcd
 
-The restore procedure for etcd configuration files replaces the appropriate
-files, then restarts the service or static pod.
+== Restoring the etcd configuration file
```

Comment:
@geliu2016 / @hexfusion 

I have restoring the etcd config file as a separate section, but let me know if it should all be incorporated to the same procedure of restoring etcd.